src/ray/rpc/worker/core_worker_server.h

```diff
@@ -114,7 +111,8 @@ class CoreWorkerGrpcService : public GrpcService {
 
   void InitServerCallFactories(
       const std::unique_ptr<grpc::ServerCompletionQueue> &cq,
-      std::vector<std::unique_ptr<ServerCallFactory>> *server_call_factories) override {
+      std::vector<std::unique_ptr<ServerCallFactory>> *server_call_factories,
+      ClusterID const *const cluster_id) override {
```

Comment:
nit. But I think we should just pass by value. By default assign cluster_id = ClusterID::Nil